python/src/EnumerateFunctionImplementation_doc.i.in

```diff
@@ -6,41 +6,45 @@ Notes
 *EnumerateFunction* represents a bijection from :math:`\Nset` to
```

Comment:
In the "See also" section, "HyperbolicEnumerateFunction" should be changed to "HyperbolicAnisotropicEnumerateFunction".

You could also add "NormInfEnumerateFunction".

Reply:
I'll change it in a new pr